api/libproject/devstart.rb

```diff
@@ -26,6 +26,10 @@
 DRY_RUN_CMD = %W{echo [DRY_RUN]}
 
 ENVIRONMENTS = {
+  "local" => {
+    :config_json => "config_local.json",
+    :cdr_versions_json => "cdr_versions_local.json"
+  },
   "all-of-us-workbench-test" => {
```

Comment:
Side comment: it might be cleaner to eventually refer to all environments by keyword, and include "project_id" as a field here. That might help reduce some conditional logic when dealing with 'local', and make it easier if we (heaven forbid) have to change a project name in the future.

Reply:
Yes, I think there needs to be a refactoring  across commands here - added a TODO.